hera_sim/visibilities/simulators.py

```diff
@@ -18,7 +18,7 @@
 from pyuvdata import UVBeam, UVData
 from pyuvsim import BeamList
 from pyuvsim import __version__ as uvsimv
-from pyuvsim import analyticbeam as ab
+from pyuvdata import analytic_beam as ab
```

Comment:
Looks like you just need UniformBeam, which can be directly imported from pyuvdata (`from pyuvdata import UniformBeam`)

Reply:
Nice!